### PR TITLE
Add Expression Tree using Array with test inputs

### PR DESCRIPTION
**Pull request by sandhyabhan**
<Issue number  #1636 >
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Code for expression tree using array added
- 

#### Languages Used:
- C
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by jainaman224** [comment]
Incorrect issue number.

File: Expression_Tree_Using_Array.txt [hunk 1]
```diff
@@ -0,0 +1,276 @@
+#include<stdio.h>
```
Comment (jainaman224) [inline Expression_Tree_Using_Array.txt]: Incorrect file extension.
Add file to folder.

File: Expression_Tree_Using_Array.txt [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+#include<stdlib.h>
+#define SIZE 10
+struct ExpTree
+{
+struct ExpTree *lc;
```
Comment (jainaman224) [inline Expression_Tree_Using_Array.txt]: Indent your code.

**Comment by jainaman224** [comment]
@sandhyabhan Are you working on it?

**Comment by sandhyabhan** [comment]
Actually I noticed that i have directly made PR without raising a issue for this code. This issue is not assigned to me as i did not raise it initially.

So Since issues from participants is not allowed.. I cannot work on this code as i don't have a issue number.

I can work on this issue if you assign this to me.

**Pull request closed without merging**